stylesheets/main.css

```diff
@@ -28,6 +28,99 @@ div#landing-header h1 {
   text-align: right;
 }
 
+#container {
```

Comment:
@anitasunshine - The `main.css` file is getting pretty big. Can we move the CSS related to the slideshow to a separate file, ex. `stylesheets/slideshow.css`, then add another `link` in `index.html` to this new CSS file?

Reply:
I'm mostly indifferent to this suggestion, but leaning slightly against since I'd rather reduce the number of request cycles (I noticed gh-pages is using HTTP 1.1).

Reply:
Good point. I think it is okay given that there are only 2 CSS files so far, but will be more of a problem if we decide to break up main.css into more files.

At some point I think we can use a CSS preprocessor like SASS to compile a single minified CSS file for production, while maintaining separate files during development.